Healthy/Classes/Modules/SavedRecipes/SavedRecipesViewController.swift

```diff
@@ -33,7 +48,42 @@ extension SavedRecipesViewController {}
 
 // MARK: - Configurations
 
-extension SavedRecipesViewController {}
+extension SavedRecipesViewController {
+    func configureNavigationBar() {
+        navigationBar.topItem?.title = "Saved recipes"
+        if let font = UIFont(name: "Poppins-Bold", size: 18) {
+            UINavigationBar.appearance().titleTextAttributes = [NSAttributedString.Key.font: font]
+        }
+    }
+
+    func configureTableView() {
+       tableView.registerNib(cell: SavedRecipesTableViewCell.self)
+    }
+
+    func configureTableViewDataSource() {
+    dataSource = UITableViewDiffableDataSource<Section, SavedRecipe>(tableView: tableView) { tableView, _, viewModel in
+        let cell = tableView.dequeueReusableCell(withIdentifier: SavedRecipesTableViewCell.reuseIdentifier)
+            as? SavedRecipesTableViewCell
+            cell?.update(with: viewModel)
+            return cell
+        }
+    }
+
+    func updateTableData(viewModel: [SavedRecipe]) {
```

Comment:
```suggestion
    func updateTableData(viewModel: [SavedRecipesTableViewCell.ViewModel]) {
```

Reply:
Model SavedRecipe Conforms to Hashable & Equatable and the
 static func == (lhs: SavedRecipe, rhs: SavedRecipe) -> Bool {
        return  lhs.title == rhs.title &&
        lhs.rating == rhs.rating &&
        lhs.chefName == rhs.chefName &&
        lhs.cookingTime == rhs.cookingTime &&
        lhs.recipeImage == rhs.recipeImage
    }
is required in unit testing to test assertEqual of obj

Reply:
This has not been applied, please recreate another pull request and address this comment.

@esraakhaled